feat: [IDP-2562] :  Added the docs on UI entity picker (#6153)
harness/developer-hub: docs/internal-developer-portal/flows/custom-extensions.md

---

> title: Supported Custom Extensions

Reviewer: ```suggestion
title: Supported Workflow UI Pickers (Field Extensions)
```

> description: These Custom Extensions are supported in Harness IDP

Reviewer: "Custom Extensions" don't mean much :)

```suggestion
description: This page lists all the Workflow UI Field Extensions that are supported in Harness IDP, which can be used as the Workflow form UI pickers.
```

sidebar_position: 5
---

> ## Custom Field Extensions

Reviewer: ```suggestion
## Workflow UI Field Extensions
```

Collecting input from the user is a very large part of the scaffolding process and Software Templates as a whole. Sometimes the built in components and fields just aren't good enough, and sometimes you want to enrich the form that the users sees with better inputs that fit better.

> This is where Custom Field Extensions come in.

Reviewer: ```suggestion
This is where Workflow UI Field Extensions come in.
```

> ### OwnerPicker

Reviewer: ```suggestion
### OwnerPicker

`OwnerPicker` is used for developers to pick a User Group from the list of Groups that exist in the account.
```
